playground/src/main/java/org/navgurukul/playground/ui/PythonPlaygroundActivity.kt

```diff
@@ -368,6 +382,200 @@ class PythonPlaygroundActivity : AppCompatActivity() {
+                    .bold { append("iterator items") }
+                errorTextExample.setText(exampleText)
+            } else if (errorText.contains("valueerror")) {//5
+                errorImage.setImageResource(R.drawable.ic_valueerror)
+
+
+                // Suppose id = 1111 and name = neil (just what you want).
+                val exampleText = SpannableStringBuilder()
+                    .append("ValueError is thrown when a function's")
+                    .bold { append("argument") }
+                    .append("is of an inappropriate type")
+                errorTextExample.setText(exampleText)
+            } else if (errorText.contains("typerror")) {//6
+                errorImage.setImageResource(R.drawable.ic_typeerror)
+
+
+                // Suppose id = 1111 and name = neil (just what you want).
+                val exampleText = SpannableStringBuilder()
+                    .append("You can't do mathematical operations on two")
+                    .bold { append("different data types.") }
+                errorTextExample.setText(exampleText)
+
+
+                val tipText = SpannableStringBuilder()
+                    .bold { append("FIX: ") }
+                    .append(" 1. Understand the datatypes of both the values/variables\n")
+                    .append(" 2. Make the data types same as each other")
+
+                errorTextTip.setText(tipText)
+            } else if (errorText.contains("nameerror")) {//7
+                errorImage.setImageResource(R.drawable.ic_nameerror)
+
+
+                // Suppose id = 1111 and name = neil (just what you want).
+                val exampleText = SpannableStringBuilder()
+                    .append("NameError is thrown when an")
+                    .bold { append("object could not be found") }
+                errorTextExample.setText(exampleText)
+            } else if (errorText.contains("keyerror")) {//8
+                errorImage.setImageResource(R.drawable.ic_keyerror)
+
+
+                // Suppose id = 1111 and name = neil (just what you want).
+                val exampleText = SpannableStringBuilder()
+                    .append("You're trying to access an key which is")
+                    .bold { append("not exisiting") }
+                    .append("in the")
+                    .bold { append("dictionary(dict).") }
+                errorTextExample.setText(exampleText)
+            } else if (errorText.contains("keyboardinterrupt")) {//9
+                errorImage.setImageResource(R.drawable.ic_valueerror)
+
+
+                // Suppose id = 1111 and name = neil (just what you want).
+                val exampleText = SpannableStringBuilder()
+                    .append("KeyboardInterrupt is thrown when the user hits")
+                    .bold { append("the interrupt key") }
+                    .append("(normally Control-C) during the execution of the program.")
+                errorTextExample.setText(exampleText)
```

Comment:
there should be a consistent logic to build error UI. Building custom UI for different errors is not scalable. Ideally we should have mapping of different resources(title, subtitle, image etc) for different errors, this whole logic should be a replaced with picking the mapped resource and updating them